lms/djangoapps/labster_course_license/models.py

```diff
@@ -17,26 +22,46 @@ class CCXLocatorField(OpaqueKeyField):
 
 class CourseLicense(models.Model):
     """
-    A Labster License.
+    A Labster License with related simulations for course.
     """
-    course_id = CCXLocatorField(max_length=255, db_index=True, unique=True)
+    course_id = CCXLocatorField(max_length=255, db_index=True, unique=True, help_text="CCXCourse locators only")
     license_code = models.CharField(max_length=255, db_index=True)
+    simulations = JSONField(help_text="List of course licensed simulations, stored as json string value")
+    created_at = models.DateTimeField(auto_now_add=True, null=True)
+    modified_at = models.DateTimeField(auto_now=True, null=True)
 
     @classmethod
     def get_license(cls, course_id):
+        """
+        Return license by course key.
+        """
         try:
             return cls.objects.get(course_id=course_id)
-        except cls.DoesNotExist:
+        except ObjectDoesNotExist:
             return None
 
     @classmethod
     def set_license(cls, course_id, license_code):
-        try:
-            course_license = cls.objects.get(course_id=course_id)
-            course_license.license_code = license_code
-        except cls.DoesNotExist:
-            course_license = cls.objects.create(course_id=course_id, license_code=license_code)
-        course_license.save()
+        """
+        Save course license.
+        """
+        course_license, __ = cls.objects.get_or_create(
```

Comment:
It doesn't update the license_code if the `course_license` exists already so please add the code below:
```
course_license, created = cls.objects.get_or_create(...)
if not created:
    course_license.license_code = license_code
    course_license.save()
```

Also please add the test for this